e2e-tests/setup.cfg

```diff
@@ -1,4 +1,13 @@
+[flake8]
+ignore = E501
```

Comment:
We're no longer running flake8 from this location, so we can remove this section.